faithcomesbyhearing/dbp
# FCBH-2695 API Key Request Submit form

# Description

* Create views for dbp admin login

## Issue Link


[FCBH-2695](https://fullstacklabs.atlassian.net/browse/FCBH-2695)

## How Do I QA This
* visit {dbp-local}/key/request
* use the form, check the modal of the terms

**Note:** To run the test suite refer to the Readme.md
